FluentUI.Demo/src/main/res/values/strings.xml

```diff
@@ -99,6 +99,13 @@
     <!-- UI label denoting the avatar view clicked action performed by the user at a specific index-->
     <string name="avatar_group_avatar_clicked">AvatarView at index %d clicked</string>
 
+    <!--Badge-->
+    <!--UI Labels -->
+    <string name="badge_notification_badge">Notification Badge</string>
+    <string name="badge_notification_dot">Dot</string>
+    <string name="badge_notification_list">List</string>
+    <string name="badge_notification_character">Character</string>
```

Comment:
please add comments to the string to clarify the meaning. Example "Character" here can be a letter or a person. A small explanation will help in translation.

Reply:
Added comments